src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/delete/DeleteTravelPlanCommand.java

```diff
@@ -11,6 +11,9 @@
 import seedu.address.model.travelplan.TravelPlan;
 import seedu.address.model.travelplanner.Model;
 
+/**
+ * Deletes a travel plan in the travel planner identified using the index from the travel planner list.
```

Comment:
The Ui didn't show the index actually, do you think we wanna show it? Then, need add into Ui part

Reply:
ok

Reply:
i think we should show index in ui